distributed-commandbus-springcloud/src/test/java/org/axonframework/springcloud/commandhandling/SpringCloudCommandRouterTest.java

```diff
@@ -110,11 +110,13 @@ public void setUp() throws Exception {
 
         when(routingStrategy.getRoutingKey(any())).thenReturn(ROUTING_KEY);
 
-        testSubject = new SpringCloudCommandRouter(discoveryClient,
-                                                   localServiceInstance,
-                                                   routingStrategy,
-                                                   s -> true,
-                                                   consistentHashChangeListener);
+        testSubject = SpringCloudCommandRouter.builder()
+                                              .discoveryClient(discoveryClient)
+                                              .localServiceInstance(localServiceInstance)
+                                              .routingStrategy(routingStrategy)
+                                              .serviceInstanceFilter(s -> true)
```

Comment:
nit: could you change this `s` to `serviceInstance` as well? As in line 337/378

Reply:
Agreed, done.